Gemfile.lock

```diff
@@ -901,5 +911,8 @@ DEPENDENCIES
   whenever
   wpcc (= 2.2.0)
 
+RUBY VERSION
+   ruby 2.5.3p105
```

Comment:
is the PATCH related with the issue when running it on MAC OSX?

Reply:
No, I don't think so - that's just the expected patch level for the 2.5.3 release. Doesn't have any bearing on the segfault issue, AFAIK.

Reply:
@scruti your question got me wondering what the point of these patch levels is, given that Ruby switched to a form of semantic version a while ago. Found this, which I think is still accurate (see 'Version schema'): https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/